plugins/services/src/js/utils/__tests__/InstanceUtil-test.js

```diff
@@ -0,0 +1,57 @@
+jest.mock("#SRC/js/structs/CompositeState");
+
+const Node = require("#SRC/js/structs/Node");
+const NodesList = require("#SRC/js/structs/NodesList");
+const CompositeState = require("#SRC/js/structs/CompositeState");
+
+const InstanceUtil = require("../InstanceUtil");
+const MasterNodeLocal = require("./fixtures/MasterNodeLocal.json");
+const MasterNodeOffsite = require("./fixtures/MasterNodeOffsite.json");
+const SlaveNodes = require("./fixtures/SlaveNodes.json");
+const NodeInstance = require("./fixtures/NodeInstance.json");
+
+describe("InstanceUtil", function() {
+  describe("#getRegionName", function() {
+    beforeEach(function() {
+      CompositeState.getNodesList = () => {
+        return new NodesList({ items: SlaveNodes });
+      };
+      CompositeState.getMasterNode = () => {
+        return new Node(MasterNodeLocal);
+      };
+    });
+    it("returns N/A when no region name exists", function() {
```

Comment:
Can we add a small test that it doesn't horribly crash when called with null? As we have no type system I would feel a bit safer with that 🙈

Reply:
done